frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -222,7 +227,7 @@ export function FieldValuesWidgetInner({
         (field, index) =>
           results[index]?.payload?.values ??
           Fields.selectors.getFieldValues(results[index]?.payload, {
-            entityId: field.id,
+            entityId: field.getUniqueId(),
```

Comment:
but store contains both `id` based and `uniqueId` based values:
- `id` is a number
- `uniqueId` is either `id` or a construction like `card__123:id`, so it's a UI specific id